src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java

```diff
@@ -86,15 +87,14 @@ public CompilerInterface() {
         add(consoleOutput, panelConstraint);
         buttonsCompiler.getRunButton().addActionListener(new ActionListener() {
             @Override
-            public void actionPerformed(ActionEvent e) {
+            public void actionPerformed(final ActionEvent e) {
                 LOGGER.info("start");
                 String url = "";
-                if (!langButtons.getPython().isEnabled()){
+                if (!langButtons.getPython().isEnabled()) {
                     url = "http://localhost:8080/api/compilePython";
                 }
                 if (!langButtons.getJava().isEnabled()) {
                     url = "http://localhost:8080/api/compileJava";
```

Comment:
Url hardcoded, could be set as a final attribute or be declared in a properties file.

Reply:
changed it.